parsexml.py

```diff
@@ -28,9 +28,13 @@ def processFile(fname):
             #get document id from metadata xml file
             for x in doc['main']['DATA_RECORD']:
                 value = x.get('DOCUMENT_IMAGE_ID',x.get('DOCUMENT_NM'))
-                with open(os.path.join(os.path.dirname(fn),'PDF_image',value+'.txt')) as dr:
-                    text = dr.read()
-                    x['DOCUMENT_TEXT'] = text
+                if os.path.isfile(os.path.join(os.path.dirname(fn),'PDF_image',value+'.txt')):
```

Comment:
I would set a variable to the path of the txt file so you don't have to repeat that code in the `if` and the `with`.

    txtfile = os.path.join(os.path.dirname(fn),'PDF_image',value+'.txt')